### PR TITLE
Improve local development

### PR DESCRIPTION
**Pull request by mjgiarlo**
This commit aims to improve local development of the media player in particular, by running a Stacks container out of a sibling directory and by serving files without requiring a media server (such as Wowza).

Closes #1855

Includes the following changes:

* Add a rake task to pull downloadable files from prod stacks into the local fake stacks root
* Ignore the fake stacks root (in `public/file/`)
* Use the druid-tools gem to construct the directory structure Stacks expects to serve files from (only in development bundler group)
* Pull in stacks' docker-compose so we can run it as part of sul-embed's `bin/dev` and mount the fake stacks root as a volume in the container
* Update README
* Add development configuration as a file

NOTE: Depends on https://github.com/sul-dlss/stacks/pull/1053, and on having an up-to-date checkout of stacks in a sibling directory. We can get rid of this in the future if we publish the stacks image.

### CODE REVIEW

File: app/components/embed/media_tag_component.rb [hunk 1]
```diff
@@ -109,6 +109,8 @@ def render_captions?
     end
 
     def streaming_settings_for(streaming_type)
+      return { mimetype: file.mimetype } if Rails.env.development?
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply app/components/embed/media_tag_component.rb:112]: Something like "Match the static file's mime type as we're circumventing the streaming server"

[the remark this answers is not in the document]
Comment (mjgiarlo) [reply app/components/embed/media_tag_component.rb:112]: You bet, thanks!

File: Procfile.dev [hunk 2]
```diff
@@ -1,3 +1,4 @@
 web: unset PORT && bin/rdbg -O -n -c -- bin/rails server
-css: yarn build:css --watch
+css: bin/yarn build:css --watch
 js: bin/shakapacker-dev-server
+stacks: docker compose up --abort-on-container-exit --build
```
Comment (jcoyne) [inline Procfile.dev:4]: Does this force a docker dependency for developers using `bin/dev`?

**Review by jcoyne — APPROVED** [review]
Approved considering comments.

File: Procfile.dev (hunk 2 above)
Comment (mjgiarlo) [reply Procfile.dev:4]: @jcoyne It does. I'll tweak that.

**Pull request merged**